src/Components/NavBar/Menus/__tests__/NewMobileNavMenu.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { SystemContextProvider } from "Artsy"
+import { mount } from "enzyme"
+import React from "react"
+
+import { menuData, SimpleLinkData } from "../../menuData"
+import { MobileLink } from "../MobileLink"
+import {
+  AnimatingMenuWrapper,
+  MobileSubmenuLink,
+  NewMobileNavMenu,
+} from "../NewMobileNavMenu"
+
+describe("MobileNavMenu", () => {
+  const getWrapper = props => {
+    return mount(
+      <SystemContextProvider user={props.user}>
+        <NewMobileNavMenu isOpen menuData={menuData} />
+      </SystemContextProvider>
+    )
+  }
+
+  describe("nav structure", () => {
+    it("renders the correct items when logged out", () => {
```

Comment:
#minor but this test is pretty long. I think it would be easier to read if it were broken into multiple `it` blocks to capture each of the cases.

Doesn't have to be for this PR!

Reply:
Yeah I'll address this in a separate PR.